xbmc/utils/ActorProtocol.cpp

```diff
@@ -241,15 +241,13 @@ bool Protocol::SendOutMessageSync(int signal, Message **retMsg, int timeout, voi
 
   if (!msg->event->WaitMSec(timeout))
   {
-    msg->origin.Lock();
```

Comment:
why do you think the lock is not required here?

Reply:
Oops! I don't think so - apparently an editor operating error. Will fix.